src/components/Icons/PhoneIcon.tsx

```diff
@@ -2,8 +2,12 @@ import React from "react";
 import Icon from "@/components/Icons/Icon";
 import { faPhone } from "@fortawesome/free-solid-svg-icons";
 
-const PhoneIcon: React.FC<{}> = () => {
-    return <Icon icon={faPhone} onHoverText="Requires follow-up phone call" color="black" />;
+interface Props {
+    color?: string;
+}
+
+const PhoneIcon: React.FC<Props> = ({ color = "black" }) => {
```

Comment:
Should we set the default colour here or rely on the one in Icon / whatever FA defaults to?

Reply:
Yeah, it's probably best to rely on the FA defaults. I've changed it now